src/pages/signup.jsx

```
@@ -0,0 +1,69 @@
+import React from "react";import { Carousel } from "react-responsive-carousel";
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)